src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -369,6 +412,111 @@ export class InputTimePicker
   //
   // --------------------------------------------------------------------------
 
+  private delocalizeTimeString(value: string): string {
+    const locale = this.effectiveLocale.toLowerCase();
+    let localeConfig,
+      valueToParse = value;
+
+    if (locale === "ar") {
```

Comment:
Can this be moved to a separate module along with the date/time locale data fetching?